KA-Flocking/Assets/Scripts/FlockAgent.cs

```diff
@@ -22,12 +22,16 @@ void Start()
         rb = GetComponent<Rigidbody>();
     }
 
-    public void Move(Vector3 velocity)
+    public void Move(Vector3 acceleration)
```

Comment:
I get this error when I create units:
NullReferenceException: Object reference not set to an instance of an object
FlockAgent.Move (UnityEngine.Vector3 acceleration) (at Assets/Scripts/FlockAgent.cs:29)
Flock.Update () (at Assets/Scripts/Flock.cs:52)